Sep21 update

tw04F opened the pull request:
pull req attempt

tw04F committed:
sep21 update

rrimando commented on home.html:
```diff
@@ -4,59 +4,60 @@
+							<h5 class="ms-3 my-2 pb-3">actions</h5>
+						</tr>
+						<tr class="lim1 text-center">
+							<td><a href=""><img src="img/fill.png" class="yelbut"></div></a></td>
```
When adding in blank anchors -- point to # `<a href="#"></a>`

rrimando reviewed (commented):
Login.html has a missing image

rrimando commented on deadcodes.html:
```diff
@@ -0,0 +1,32 @@
+		<!--<div class="col-6 cc2 ct0 rounder my-3 d-inline-block">
```
There is a file called eto na muna and dead codes kindly rename them as necessary - it may be quite confusing when we invite other people to this codebase

rrimando commented on home.html:
```diff
@@ -79,7 +80,7 @@ <h5 class="ms-3 my-2 pb-3">actions</h5>
 							</div></a>
 						<a href="" class="ct0">
-							<div class="button blop cc2 rounder me-5 py-2">TEST CAMPAIGNS<br />
+							<div class="button blop cc2 rounder me-2 py-2">TEST CAMPAIGNS<br />
```
There seems to be 4 empty png files here - libg, png1, png2 and png3. I am unsure what they are for.

rrimando commented on css/style.css:
```diff
@@ -123,16 +123,11 @@ input[type="password"]
 
 .custcont1 
```
I suggest using naming conventions such as - custom-content-one - instead of shortening it. While it works perfectly fine - being more explicit makes your code more readable.

rrimando commented on css/style.css:
```diff
@@ -181,15 +179,16 @@ input[type="password"]
+	border-radius: 100%;
 	background-color: yellow;
 }
 
```
Install HTML prettify plugin and auto format your CSS

rrimando commented on css/style.css:
```diff
@@ -181,15 +179,16 @@ input[type="password"]
+	height: auto;
+	min-width: 50px;
+	border-radius: 100%;
 	background-color: yellow;
```
Use hexcodes for colors - you can install colorpicker extension plugin on your browser

rrimando commented on home.html:
```diff
@@ -4,59 +4,60 @@
+				<div class="custcont1 lim2">
+					<table class="table">
+						<tr><td>seo leads</td></tr>
+						<tr class="ct1 cc2">
```
Kindly capitalize these text blocks to make them appear more formal

tw04F committed:
8:21 - 9.21 Hotfix

tw04F committed:
11:08PM SEP-21

Didn't have much progress, got stuck trying to style a custom bullet and making the page responsive
